### ISSUE TITLE
There are more places in other docs too where this is note is not updated...

### ISSUE THREAD

**Issue by parth-gr** [rook/rook#10458]
There are more places in other docs too where this is not updated...
Maybe we can open an issue for it.

_Originally posted by @parth-gr in https://github.com/rook/rook/pull/10234#discussion_r898109037_

**Comment by travisn** [comment]
@parth-gr Could you summarize the issue in the title and body instead of just adding the link? thanks

**Comment by parth-gr** [comment]
Like the way of mentioning a note has been changed,

**note** , > note to !!! note

So some places are missed to be updated, like https://github.com/rook/rook/blame/master/design/ceph/interacting-with-rook-resources.md#L43, https://github.com/rook/rook/blame/master/design/ceph/ceph-config-updates.md#L138 , etc

**Comment by YashasviChaurasia** [comment]
@parth-gr @travisn @hunter I would like to work on this issue.
Thank you.
